cli/src/ios/update.ts

```diff
@@ -103,6 +104,9 @@ async function updatePodfile(
   );
   podfileContent = podfileContent.replace(
     `require_relative '../../node_modules/@capacitor/ios/scripts/pods_helpers'`,
```

Comment:
this should be a regular expression as the users might have changed the path already and might need adjustments later on